### ISSUE TITLE
DeprecationWarning: BaseException.message has been deprecated as of Python 2.6

### ISSUE THREAD

**Issue by TNTBOMBOM** [QubesOS/qubes-issues#4519]
### Qubes OS version:

Qubes 4

### Affected component(s): 

Installing Whonix Templates

---

### Steps to reproduce the behavior:

````
[user@dom0 ~]$ sudo qubesctl state.sls qvm.anon-whonix
[WARNING ] /var/cache/salt/minion/extmods/states/ext_state_qvm.py:142: DeprecationWarning: BaseException.message has been deprecated as of Python 2.6
  status = Status(retcode=1, result=False, stderr=err.message + '\n')
 
[ERROR   ] ====== ['features'] ======
Virtual Machine does not exist!
 
====== ['tags'] ======
[SKIP] Skipping due to previous failure!
[ERROR   ] ====== ['features'] ======
Virtual Machine does not exist!
 
====== ['tags'] ======
[SKIP] Skipping due to previous failure!
[ERROR   ] ====== ['present'] ======
====== stderr ======
/usr/bin/qvm-create sys-whonix --class=AppVM --template=whonix-gw-14 --label=black --property=memory=500
app: Error creating VM: Got empty response from qubesd. See journalctl in dom0 for details.
 
====== ['prefs'] ======
Virtual Machine does not exist!
local:
----------
          ID: template-whonix-ws-14
    Function: pkg.installed
        Name: qubes-template-whonix-ws-14
      Result: True
     Comment: Package qubes-template-whonix-ws-14 is already installed
     Started: 06:47:58.936426
    Duration: 2819.299 ms
     Changes:  
----------
          ID: whonix-ws-tag
    Function: qvm.vm
        Name: whonix-ws-14
      Result: False
     Comment: ====== ['features'] ======
              Virtual Machine does not exist!
             
              ====== ['tags'] ======
              [SKIP] Skipping due to previous failure!
     Started: 06:48:01.757710
    Duration: 14.668 ms
     Changes:  
----------
          ID: whonix-ws-update-policy
    Function: file.prepend
        Name: /etc/qubes-rpc/policy/qubes.UpdatesProxy
      Result: True
     Comment: File /etc/qubes-rpc/policy/qubes.UpdatesProxy is in correct state
     Started: 06:48:01.776031
    Duration: 3.64 ms
     Changes:  
----------
          ID: whonix-get-date-policy
    Function: file.prepend
        Name: /etc/qubes-rpc/policy/qubes.GetDate
      Result: True
     Comment: File /etc/qubes-rpc/policy/qubes.GetDate is in correct state
     Started: 06:48:01.779786
    Duration: 1.525 ms
     Changes:  
----------
          ID: template-whonix-gw-14
    Function: pkg.installed
        Name: qubes-template-whonix-gw-14
      Result: True
     Comment: Package qubes-template-whonix-gw-14 is already installed
     Started: 06:48:01.781416
    Duration: 0.423 ms
     Changes:  
----------
          ID: whonix-gw-tag
    Function: qvm.vm
        Name: whonix-gw-14
      Result: False
     Comment: ====== ['features'] ======
              Virtual Machine does not exist!
             
              ====== ['tags'] ======
              [SKIP] Skipping due to previous failure!
     Started: 06:48:01.781930
    Duration: 13.077 ms
     Changes:  
----------
          ID: whonix-gw-update-policy
    Function: file.prepend
        Name: /etc/qubes-rpc/policy/qubes.UpdatesProxy
      Result: True
     Comment: File /etc/qubes-rpc/policy/qubes.UpdatesProxy is in correct state
     Started: 06:48:01.795128
    Duration: 1.848 ms
     Changes:  
----------
          ID: sys-net
    Function: qvm.exists
      Result: True
     Comment: /usr/bin/qvm-check sys-net
              VM sys-net exists None
     Started: 06:48:01.797083
    Duration: 261.526 ms
     Changes:  
----------
          ID: sys-firewall
    Function: qvm.exists
      Result: True
     Comment: /usr/bin/qvm-check sys-firewall
              VM sys-firewall exists None
     Started: 06:48:02.059109
    Duration: 273.134 ms
     Changes:  
----------
          ID: sys-whonix
    Function: qvm.vm
      Result: False
     Comment: ====== ['present'] ======
              ====== stderr ======
              /usr/bin/qvm-create sys-whonix --class=AppVM --template=whonix-gw-14 --label=black --property=memory=500
              app: Error creating VM: Got empty response from qubesd. See journalctl in dom0 for details.
             
              ====== ['prefs'] ======
              Virtual Machine does not exist!
     Started: 06:48:02.334414
    Duration: 759.933 ms
     Changes:  
----------
          ID: whonix-ws-14-dvm
    Function: qvm.vm
      Result: False
     Comment: One or more requisite failed: qvm.sys-whonix.sys-whonix
     Changes:  
----------
          ID: qvm-appmenus --update whonix-ws-14-dvm
    Function: cmd.run
      Result: False
     Comment: One or more requisite failed: qvm.whonix-ws-dvm.whonix-ws-14-dvm
     Changes:  
----------
          ID: anon-whonix
    Function: qvm.vm
      Result: False
     Comment: One or more requisite failed: qvm.sys-whonix.sys-whonix, qvm.whonix-ws-dvm.whonix-ws-14-dvm
     Changes:  
 
Summary for local
------------
Succeeded: 7
Failed:    6
------------
Total states run:    13
Total run time:   4.149 s
DOM0 configuration failed, not continuing
````

### Expected behavior:

to install whonix softly and smoothly 

### Actual behavior:

not happening

### General notes:

- This has been reported by one of our users on IRC channel (djote)
- Not sure if the title describing the actual problem

**Comment by djote63** [comment]
I reported the issue in irc, thanks TNTBOMBOM :)

**Comment by marmarek** [comment]
BTW the issue title have very little to do with actual problem. But is also something to fix.

**Comment by TNTBOMBOM** [comment]
@marmarek  i already mentioned that on the same topic , please feel free to change it to the suitable name.

**Comment by djote63** [comment]
Thanks marmarek, unfortunately didnt solve the issue. https://pastebin.com/ws55gd2U

**Comment by marmarek** [comment]
Oh, removing template _package_ fails, when template _VM_ is already removed... Try `sudo rpm -e --noscripts qubes-template-whonix-ws-14` - this should avoid checking for template _VM_ during the operation.

**Comment by northox** [comment]
Just to let you know that @marmarek's proposal fixed it for me. ` sudo rpm -e --noscripts qubes-template-whonix-ws-14`

**Comment by DemiMarie** [comment]
Is this even relevant anymore?  Can this issue be closed now?

**Comment by TNTBOMBOM** [comment]
Im not sure its fixed because there is another ticket on #620 having the 
same problem.

(Unless there are patches already been pushed to fix this issue then 
feel free to close it)

**Comment by TNTBOMBOM** [comment]
Do you see it fixed? because there is another ticket on #620 having the 
same problem.

(Unless there are patches already been pushed to fix this issue then 
feel free to close it)
